ui/src/app/pages/admin/admin-institution-options.tsx

```diff
@@ -0,0 +1,12 @@
+import {DuaType, OrganizationType} from 'generated/fetch';
+
+export const DuaTypes =  [{label: 'Master', value: DuaType.MASTER},
+  {label: 'Individual', value: DuaType.RESTRICTED}];
+
+export const OrganizationTypeOptions = [
+  {label: 'Industry', value: OrganizationType.INDUSTRY},
+  {label: 'Academic Research Institution', value: OrganizationType.ACADEMICRESEARCHINSTITUTION},
+  {label: 'Educational Institution', value: OrganizationType.EDUCATIONALINSTITUTION},
+  {label: 'Health Center/ Non-Profit', value: OrganizationType.HEALTHCENTERNONPROFIT},
```

Comment:
nitpick: consistent spacing around the slash.  Either 'Health Center/Non-Profit' or 'Health Center / Non-Profit'